microceph/ceph/keyring.go

```diff
@@ -106,3 +106,39 @@ func parseKeyring(path string) (string, error) {
 
 	return cephSecret, nil
 }
+
+// CreateKey creates a client key and returns the said key hash without saving it as a file.
+var CreateKey = func(clientName string, caps ...[]string) (string, error) {
```

Comment:
See above re: mocking. The `cephRun()` func you're using here to perform system commands can be mocked already, see `var processExec Runner = RunnerImpl{}`